frontend/src/metabase/css/core/colors.module.css

```diff
@@ -98,60 +103,13 @@
 
   --mb-base-color-white: #fff;
 
-  /* Brand */
-  --mb-base-color-brand-70: color-mix(
-    in srgb,
-    var(--mb-color-brand),
-    black 50%
-  );
-  --mb-base-color-brand-60: color-mix(
-    in srgb,
-    var(--mb-color-brand),
-    black 25%
-  );
-  /*   brand */
-  --mb-base-color-brand-40: var(--mb-color-brand);
-  --mb-base-color-brand-30: color-mix(
-    in srgb,
-    var(--mb-color-brand),
-    white 45%
-  );
-  --mb-base-color-brand-20: color-mix(
-    in srgb,
-    var(--mb-color-brand),
-    white 70%
-  );
-  --mb-base-color-brand-10: color-mix(
-    in srgb,
-    var(--mb-color-brand),
-    white 90%
-  );
-
-  /* @deprecated Blue */
-  --mb-base-color-blue-100: #00172b;
-  --mb-base-color-blue-90: #042746;
-  --mb-base-color-blue-80: #0a3b67;
-  --mb-base-color-blue-70: #105490;
-  --mb-base-color-blue-60: #1871bf;
-  --mb-base-color-blue-50: #358cd9;
-  --mb-base-color-blue-40: #509ee3; /* brand */
-  --mb-base-color-blue-30: #8dc0ed;
-  --mb-base-color-blue-20: #cbe2f7; /* focus */
-  --mb-base-color-blue-10: #eef6fd; /* baby blue */
-  --mb-base-color-blue-5: #f8fbfe;
-
-  /* @deprecated Gray */
-  --mb-base-color-gray-100: #121619; /* bg-black */
-  --mb-base-color-gray-90: #20272b;
-  --mb-base-color-gray-80: #313b42;
-  --mb-base-color-gray-70: #47545e;
-  --mb-base-color-gray-60: #657079;
-  --mb-base-color-gray-50: #808991;
-  --mb-base-color-gray-40: #929aa1;
-  --mb-base-color-gray-30: #b4bcc3;
-  --mb-base-color-gray-20: #dbdfe3;
-  --mb-base-color-gray-10: #f3f5f7;
-  --mb-base-color-gray-5: #fafbfc;
+  /* Brand - can be overridden by whitelabeling */
```

Comment:
as you mentioned here, we probably need a better color solution for automatic brand color. Since brand color could be set to any lightness level. I couldn't think of a simple way to do this yet. I feel like this is one of the place where we really need some JavaScript logic to inject these variables.

First, we need to detect the base lightness of the set `brand` color, then populate the rest colors in the palette.

We might also need to adjust hover state accordingly. Since currently it's static e.g. base of 40 and we set the hover to 10 that's 30 difference and is enough for the contrast, but if the background is in any different lightness because of the set brand color we shouldn't be hard code the hover state, but rather say like either move the lightness up or down for n level or something.